### PR TITLE
[SERVICE-864] Create feed [un-]subscribe UI

### PR DESCRIPTION
**Pull request by tomrobertsOF**
- Adds a new panel, reachably through the "add a feed" button on the settings panel. 
- Contains a list of all available feeds.
- For now, the feeds list is hard coded, but as it is done through the store, it should be straightforward to wire it up to some external data source

### CODE REVIEW

File: src/provider/view/components/SettingsView/SettingsView.module.scss [hunk 1]
```diff
@@ -43,4 +43,9 @@
             border-bottom: 1px solid var(--color-light);
         }
     }
+
+    .placeholder-text {
```
Comment (TheoOttahOF) [inline src/provider/view/components/SettingsView/SettingsView.module.scss]: Can the placeholder-text be changed to something more descriptive of what it actually is?

File: src/provider/view/containers/NotificationCenterApp/NotificationsFeedsPanel/FeedsPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import {ClassNameBuilder} from '../../../utils/ClassNameBuilder';
+import {FeedsView} from '../../../components/FeedsView/FeedsView';
+
+import * as Styles from './FeedsPanel.module.scss';
+
+interface Props extends RouteComponentProps {
+
+}
+
+const FeedsPanelComponent: React.FC<Props> = (props) => {
```
Comment (TheoOttahOF) [inline src/provider/view/containers/NotificationCenterApp/NotificationsFeedsPanel/FeedsPanel.tsx]: Props should define `history: MemoryHistory`; Then you don't need to cast or use RouteComoponentProps as that is the wrong try of history.

File: src/provider/view/components/SettingsView/SettingsView.module.scss (hunk 1 above)
Comment (tomrobertsOF) [reply src/provider/view/components/SettingsView/SettingsView.module.scss]: Well what would you call it? It's the placeholder text that shows when you have no feeds

Comment (TheoOttahOF) [reply src/provider/view/components/SettingsView/SettingsView.module.scss]: empty-message/no-feeds-text?

Comment (tomrobertsOF) [reply src/provider/view/components/SettingsView/SettingsView.module.scss]: 👍

File: src/provider/view/containers/NotificationCenterApp/NotificationsFeedsPanel/FeedsPanel.tsx (hunk 2 above)
Comment (tomrobertsOF) [reply src/provider/view/containers/NotificationCenterApp/NotificationsFeedsPanel/FeedsPanel.tsx]: Though i had changed all of them, but must have missed one. Will fix now.

**Comment by tomrobertsOF** [comment]
@ryanpetersonOF, @TheoOttahOF, should be ready for a second look now.

File: src/provider/view/components/FeedsView/FeedsView.module.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.feeds-view {
+    user-select: none;
+    height: 100%;
+    background: var(--color-background-light);
+    color: var(--color-light);
+
+    .placeholder-text {
```
Comment (TheoOttahOF) [inline src/provider/view/components/FeedsView/FeedsView.module.scss]: Rename this aswell

Comment (tomrobertsOF) [reply src/provider/view/components/FeedsView/FeedsView.module.scss]: Done

**Pull request merged**